### DISCUSSION TITLE
so-setup Iso doesn't recognize Oracle

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13068] [2.4]
### Version

2.4.60

### Installation Method

Other (please provide detail below)

### Description

installation

### Installation Type

other (please provide detail below)

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

32

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Playing around with an option to install Oracle first and then pull onion from gitlab so I tried running so-setup iso and I am getting error saying that "We do not support the version of CentOS you are trying to use."

```
[root@localhost setup]# hostnamectl
   Static hostname: (unset)
Transient hostname: localhost
         Icon name: computer-vm
           Chassis: vm 🖴
        Machine ID: c3c5065631bd42778484af8039a1e53b
           Boot ID: 003e092441494f8280fcf936f419805e
    Virtualization: vmware
  Operating System: Oracle Linux Server 9.4
       CPE OS Name: cpe:/o:oracle:linux:9:4:server
            Kernel: Linux 5.15.0-206.153.7.el9uek.x86_64
      Architecture: x86-64
   Hardware Vendor: VMware, Inc.
    Hardware Model: VMware Virtual Platform
  Firmware Version: 6.00
[root@localhost setup]# bash so-setup iso
Getting started...
We do not support the version of CentOS you are trying to use.
[root@localhost setup]# pwd
/root/securityonion/setup
```

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by SliuzasLukas** [comment]
I just tried so-setup-network I got the same thing

**Comment by SliuzasLukas** [comment]
Okey for some weird reason doing git clone got me bad files? Used zipped file and it works now

**Reply by dougburks** [reply]
It looks like you cloned the old 2.3 version instead of the new 2.4 version.

Please keep in mind that network installations are NOT supported:
https://docs.securityonion.net/en/2.4/network-installation.html
